composer.json

```diff
@@ -29,11 +27,17 @@
         "psr-4": { "MongoDB\\Tests\\": "tests/" },
         "// Manually include assertion functions for PHPUnit 8.x and earlier ":"",
         "// See: https://github.com/sebastianbergmann/phpunit/issues/3746 ":"",
-        "files": [ "vendor/phpunit/phpunit/src/Framework/Assert/Functions.php" ]
+        "files": [
+            "vendor/bin/.phpunit/phpunit/src/Framework/Assert/Functions.php",
+            "tests/PHPUnit/FunctionsPolyfill.php"
+        ]
     },
     "extra": {
         "branch-alias": {
             "dev-master": "1.8.x-dev"
         }
+    },
+    "scripts": {
+        "pre-autoload-dump": "if test -f vendor/bin/simple-phpunit; then vendor/bin/simple-phpunit --version > /dev/null; fi"
```

Comment:
What's the purpose of dumping the version of `simple-phpunit` to `/dev/null`? Is this something you're instructed to do as a consumer of `simple-phpunit`? I'd also be curious if the embedded script poses a problem for Windows.

Reply:
I'll add a comment, but in order to get our autoload-dev directive to work, we need to ensure phpunit is installed, which can be accomplished by dumping the version.

This will most likely be changed once again since PHPUnit 9 moves the `assert*` functions to a different namespace and autoloads them automatically, so I'll have to create a separate polyfill for that. For now I'm just happy to finally get this working properly.

Reply:
Noted. If this is temporary, perhaps a JSON comment would be prudent (like I did for `autoload-dev`). Up to you, though.

Reply:
Murphy's law suggests that if I don't add a comment, the code would stay in for more than a few days. I'll add one ;)

Reply:
The simple-phpunit script supports `vendor/bin/simple-phpunit install` to install the corresponding PHPUnit version